generate-stackbrew-library.sh

```diff
@@ -110,21 +110,9 @@ for version; do
 		suiteAliases=( "${suiteAliases[@]//latest-/}" )
 		variantAliases+=( "${suiteAliases[@]}" )
 
-		# calculate the intersection of parent image arches and gosu arches
-		arches="$(jq -r --arg arches "$arches" '
-			(
-				$arches
-				| gsub("^[[:space:]]+|[[:space:]]+$"; "")
-				| split("[[:space:]]+"; "")
-			) as $parentArches
-			| .[env.version]
-			| $parentArches - ($parentArches - (.gosu.arches | keys))
-			| join(", ")
-		' versions.json)"
-
 		echo
 		cat <<-EOE
-			Tags: $(join ', ' "${variantAliases[@]}")
+			Tags: $(join ', ' $arches)
 			Architectures: $arches
```

Comment:
Almost :sweat_smile:

```suggestion
			Tags: $(join ', ' "${variantAliases[@]}")
			Architectures: $(join ', ' $arches)
```

Reply:
TY. Sorry for the lacking attention to detail in this block. I have not studied it enough to understand it.